jaseci_ai_kit/install.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
 JAC_NLP_MODULES=("bart_sum" "cl_summer" "ent_ext" "fast_enc" "sbert_sim" "t5_sum" "text_seg" "tfm_ner" "use_enc" "use_qa" "zs_classifier" "bi_enc" "topic_ext")
-JAC_SPEECH_MODULES=("stt" "tts" "vc_tts")
+JAC_SPEECH_MODULES=("stt" "vc_tts")
 JAC_MISC_MODULES=("pdf_ext" "translator" "cluster" "ph")
 JAC_VISION_MODULES=("detr" "rftm" "yolos" "dpt")
 
 install_modules() {
 	if [[ $1 == "all" ]]; then
 		echo -e "\033[0;32mInstalling all the modules\033[0m"
 		cd jac_speech
-		pip install -e .[all]
+		pip install .[all]
```

Comment:
Is there a reason we don't want to install live for just jac_speech here?

Reply:
When we install with -e sign the packages in the ORDEREDREQS list in setup file doesn't get installed.

Reply:
I see. In that case, I have removed `-e` for the other modules as well to keep the behavior of this script consistent.